target/scripts/helper-functions.sh

```diff
@@ -1,6 +1,8 @@
 #! /bin/bash
 
 DMS_DEBUG="${DMS_DEBUG:=0}"
+SCRIPT_NAME="$(basename "$0")"
```

Comment:
Probably add a comment that it refers to the script that sourced `helper-functions.sh`, not `helper-functions.sh`, which was believed to be the case by yourself earlier (and for maintainers like me that are shell-phobic :sweat_smile: ).

Reply:
Can do